include/odp/api/spec/ml_types.h

```diff
@@ -512,6 +512,22 @@ typedef struct odp_ml_shape_info_t {
 
 } odp_ml_shape_info_t;
 
+/**
+ * Quantization parameters
+ *
+ * These parameters are used to convert between floating point and integer data. Scale and zerop
+ * values can be used directly with the odp_ml_fp32_from_*() and odp_ml_fp32_to_*() functions. Both
+ * scale and zerop are 0, if the input or output is not quantized, or these parameters are not
+ * available for the model, or the implementation does not provide these parameters.
```

Comment:
This sentence could be simplified by removing "if the input or output is not quantized, or these parameters are not
 available for the model". Just saying that those are zero when "not supported".

Reply:
Zero is a legal and probably not an uncommon value for zerop, though. So application will anyway have to check for zero scale. So if we want to eliminate the float comparison, we should probably add a boolean to indicate whether these parameters are valid or not.

Reply:
True. I think we could add some flag(s) to odp_ml_model_info_t level to tell if quantization info is supported (or not), then the sentence can be also removed here.

Reply:
Added in v4, and the sentence replaced accordingly.